sites/svelte.dev/src/routes/repl/create.json.js

```diff
@@ -1,10 +1,11 @@
 import * as gist from '$lib/db/gist';
 
-export async function post({ locals, body }) {
+export async function post({ locals, request }) {
```

Comment:
will this need to be `event` now?

Reply:
[See other comment.](https://github.com/sveltejs/sites/pull/270#discussion_r789086200)